firebase_admin/auth.py

```diff
@@ -14,48 +14,50 @@
 
```

Comment:
Done. Extracted to a new `_user_mgt.py` module. I had to change things around a little bit to break a few cyclic dependencies.